README.md

```diff
@@ -1183,6 +1184,37 @@ sum(10, 20); // Returns value returned by the function
 sum(10, 20); // Cache hit!
 ```
 
+### [just-memoize-one](https://www.npmjs.com/package/just-memoize-one)
+
+:icecream:[`Try It`](https://anguscroll.com/just/just-memoize-one)
+
+`npm install just-memoize-one`
+
+```js
+import memoizeOne from 'just-memoize-one';
+
+const sumByOne = memoizeOne(function(value) {
+  return value + 1;
+});
+
+sumByOne(10); // Returns value returned by the function
+sumByOne(10); // Cache hit!
+
+sumByOne(20); // Returns value returned by the function
+sumByOne(20); // Cache hit!
```

Comment:
I would insert the line
`sumByOne(10); // Returns value returned by the function`
after this, to show that the previous cache has now been invalidated